HW04.Rmd

```diff
@@ -69,7 +127,22 @@ Assume that:
 * Fall = October-December
 
 ```{r season}
-
+total_month <- gun_deaths %>% 
+    select(month) %>%
+    group_by(month) %>% 
+    count()
+season <- factor(c("Winter", "Winter", "Winter", 
+                   "Spring", "Spring", "Spring",
+                   "Summer", "Summer", "Summer",
+                   "Fall", "Fall", "Fall"))
+number_season <- cbind(total_month, season)
+colnames(number_season) <- c("month", "n", "season")
+max_season <- number_season %>% 
+    group_by(season) %>% 
+    summarize(total_season = sum(n)) %>% 
+    arrange(desc(total_season))
+print(paste(max_season$season[1], "has the most gun deaths."))

```

Comment:
Again, very nice way of automatically filtering for the group with the max number of observations. I'm sure this is especially useful when working with datasets that have a ton of different groups.